arelle/plugin/validate/EFM/resources/taxonomy-compatibility.json

```diff
@@ -62,11 +62,12 @@
 
         "comment2": "latest us-gaap is compatible with prior year ifrs from Q1 to Q2, after Q2 must use same-year ifrs",
         "us-gaap/2021": ["@stx-2021", "srt/2021", "ifrs/2021", "cef/2021q4"],
-        "us-gaap/2022": ["@stx-2022", "srt/2022", "ifrs/2022", "cef/2022"],
+        "us-gaap/2022": ["us-gaap/2022q3", "@stx-2022", "srt/2022", "srt/2022q3", "ifrs/2022", "cef/2022"],
+        "us-gaap/2022q3": ["us-gaap/2022", "@stx-2022", "srt/2022q3", "srt/2022", "ifrs/2022", "cef/2022"],

```

Comment:
Is this correct?
`"us-gaap/2022"` contains `["us-gaap/2022q3",...]`
and
`"us-gaap/2022q3"` contains `["us-gaap/2022",...]`

Reply:
That is what was requested.